app/javascript/member-facing/backbone/petition.js

```diff
@@ -17,6 +17,13 @@ const Petition = Backbone.View.extend({
     this.followUpUrl = options.followUpUrl;
     this.submissionCallback = options.submissionCallback;
     this.skipOnSuccessAction = options.skipOnSuccessAction;
+
+    if (options.redirectAfterAction !== undefined) {
+      this.redirectAfterAction = options.redirectAfterAction;
+    } else {
+      this.redirectAfterAction = true;
+    }
```

Comment:
thanks, got it! What about updating the options documentation above the function. https://github.com/SumOfUs/Champaign/pull/1033/files#diff-66858a10448a660aa22d43213512f65dL13